.appveyor.yml

```diff
@@ -24,7 +24,7 @@ init:
 
 build_script:
   - cmd: 'gradlew.bat --version'
-  - cmd: 'gradlew.bat %GRADLE_CLI_OPTS% checkstyle check build'
+  - cmd: 'gradlew.bat %GRADLE_CLI_OPTS% test testStreaming testNg -xyarnSetup -xyarn -xbuildWeb -xcheckstyle -xcheckstyleMain -xcheckstyleJava9 -xcheckstyleTest -xcheckstyleJava9Test -xcheckstyleJmh'
```

Comment:
We already have a `checkstyleEnabled` local variable in `java.gradle` - how about making a one line change so it checks a property like `com.linecorp.checkstyleDisabled`?

Reply:
Now using `noCheckstyle` and also added `noWeb`